docs/integrations/github/discovery.md

```diff
@@ -38,17 +38,28 @@ The target is composed of three parts:
 
 ## GitHub API Rate Limits
 
-GitHub
-[rate limits](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting)
-API requests to 5,000 per hour (or more for Enterprise accounts). The default
-Backstage catalog backend refreshes data every 100 seconds, which issues an API
-request for each discovered location.
+GitHub [rate limits] API requests to 5,000 per hour (or more for Enterprise
+accounts). The default Backstage catalog backend refreshes data every 100
+seconds, which issues an API request for each discovered location.
 
 This means if you have more than ~140 catalog entities, you may get throttled by
 rate limiting. This will soon be resolved once catalog refreshes make use of
 ETags; to work around this in the meantime, you can change the refresh rate of
-the catalog in your `packages/backend/src/plugins/catalog.ts` file, or configure
-Backstage to use the [github-apps plugin](../../plugins/github-apps.md).
+the catalog in your `packages/backend/src/plugins/catalog.ts` file:
+
+```typescript
+const builder = await CatalogBuilder.create(env);
+
+// For example, to refresh every 5 minutes (300 seconds).
+builder.setRefreshIntervalSeconds(300);
+```
+
+Alternatively, or additionally, you can use the [github-apps plugin] which
```

Comment:
```suggestion
Alternatively, or additionally, you can configure [github-apps] authentication which
```

Reply:
Happy to change this, but wondering if "authentication" might be conflated with the [GitHub authentication provider](https://backstage.io/docs/auth/github/provider)? Perhaps "github-apps integration" is more correct? I'm not sure.

This page is also referred to as "github-apps plugin" in [this document](https://backstage.io/docs/integrations/github/locations#authentication-with-github-apps), so I'm somewhat worried about the inconsistency if I change it here.

Reply:
That wording in the other article is wrong - it's for sure not a plugin in and of itself.